tools/ci_build/github/azure-pipelines/azure-pipelines-py-packaging.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       displayName: 'Run build script'
       inputs:
         scriptPath: 'tools/ci_build/github/linux/run_dockerbuild.sh'
-        args: '-c Release -o ubuntu16.04 -d cpu -r $(Build.BinariesDirectory) -p $(python.version) -x "--build_wheel --use_mkldnn --use_tvm --test_data_url $(TestDataUrl) --test_data_checksum $(TestDataChecksum)"'
+        args: '-c Release -o ubuntu16.04 -d cpu -r $(Build.BinariesDirectory) -p $(python.version) -x "--build_wheel --use_tvm --test_data_url $(TestDataUrl) --test_data_checksum $(TestDataChecksum)"'
```

Comment:
Is there any reason to put tvm here?

Reply:
I guess we need openmp to enable parallelization. Don't we? Multiple ops take advantage of it.

As for tvm, I guess we can remove it. None of the existing code is taking advantage of tvm to the best of my knowledge.

Reply:
I agree with you. Would you do both of these changes?

Reply:
done.